spec/lib/board_spec.rb

```diff
@@ -0,0 +1,252 @@
+require_relative '../spec_helper'
+
+RSpec.describe Board do
+  describe "#initialize" do
+    it "should create two dimensional board_state array with given size parameter" do
+      board = Board.new(3)
```

Comment:
[Let is your friend ](http://www.betterspecs.org/#let)

Reply:
It's good idea, we changed it through the way but I think our case doesn't fit with it :) I know what you wanted to say.